pkg/commands/compute/build.go

```diff
@@ -151,11 +169,29 @@ func (c *BuildCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		}
 	}
 
-	if c.Manifest.File.Scripts.Build != "" {
-		lang = "custom"
+	// NOTE: A third-party could share a project with a build command for a
+	// language that wouldn't normally require one (e.g. Rust), and do evil
+	// things. So we should notify the user and confirm they would like to
+	// continue with the build.
+	if !c.Flags.AcceptCustomBuild && toolchain == "custom" {
+		customBuildMsg := "This project has a custom build script defined in the fastly.toml manifest. "
+		if c.Globals.Flag.Verbose {
+			text.Info(out, customBuildMsg)
+			customBuildMsg = ""
+		}
```

Comment:
I wonder if we should always display this in verbose mode for custom build scripts, regardless of the AcceptCustomBuild flag.

Reply:
Have fixed locally, will push up shortly